gatsby/content/docs/e2ee_cross_signing.mdx

```diff
@@ -1196,3 +1196,223 @@ and use that to verify and sign yourself:
 4. Sign your own master key with your own device key.
 5. Upload the signatures.
 6. Fetch all entries from SSSS that you want to cache and cache them.
+
```

Comment:
@Sorunome The current state of the content looks good as an iterative update. Is the PR in a draft state for a particular blocker?